.travis.yml

```diff
@@ -43,7 +43,7 @@ before_script:
   - adb shell input keyevent 82 &
 
 script:
-  - ./gradlew build jacocoTestReport assembleAndroidTest connectedCheck
+  - ./gradlew clean build jacocoTestReport assembleAndroidTest connectedCheck
```

Comment:
why clean? It will delete all build artifacts from the `before_script` `assemble` step